test/tbb/test_concurrent_lru_cache.cpp

```diff
@@ -125,3 +126,25 @@ TEST_CASE("basic test for eviction of only unused items 2") {
     REQUIRE_MESSAGE(is_correct, "cache should not evict items in use");
 }
 
+//! \brief \ref error_guessing
+TEST_CASE("basic test for handling case when number_of_lru_history_items is zero") {
+    auto counter_func = [] (int) {
+        return utils::LifeTrackableObject{};
+    };
+    using cache_type =  tbb::concurrent_lru_cache<int, utils::LifeTrackableObject, decltype(counter_func)>;
+    cache_type cache{counter_func, 0};
+    
+    for(int i = 0; i < 10; ++i) {
+        const utils::LifeTrackableObject* obj_addr = &cache[1].value();
```

Comment:
Consider commenting what is going on:

```suggestion
        const utils::LifeTrackableObject* obj_addr = &cache[1].value();
        // Destructor of the handle returned by the operator[] calls immediately
```

Reply:
Didn't noticed suggestion. But seem like mine says similar thing.